code/__DEFINES/~skyrat_defines/manufacturer_strings.dm

```diff
@@ -0,0 +1,22 @@
+// Text for what examine blurb will come up when used with /datum/element/manufacturer_examine
+// Very loosely grouped based on what company makes what mainly for your viewing pleasure
+
+#define COMPANY_CANTALAN "It has <b>[span_purple("Cantalan Federal Arms")]</b> etched into the grip."
+#define COMPANY_DONK "It has a <b>[span_green("Donk Corporation")]</b> label visible in the plastic."
+#define COMPANY_BOLT "It has <b>[span_yellow("Bolt Fabrications")]</b> stamped onto the reciever."
+#define COMPANY_NANOTRASEN "It has <b>[span_blue("Nanotrasen Armories")]</b> etched into the reciever."
+
+#define COMPANY_ARMADYNE "It has <b>[span_red("Armadyne Corporation")]</b> etched into the barrel."
+#define COMPANY_OLDARMS "It has <b><i>[span_red("Armadyne Oldarms")]</i></b> etched into the barrel."
+#define COMPANY_SCARBOROUGH "It has <b>[span_orange("Scarborough Arms")]</b> stamped onto the grip."
+#define COMPANY_INTERDYNE "It has <b>[span_cyan("Interdyne Pharmaceuticals")]</b> stamped onto the barrel."
+
+#define COMPANY_ALLSTAR "It has <b>[span_red("Allstar Lasers Inc.")]</b> stamped on the front grip."
+#define COMPANY_MICRON "It has <b>[span_cyan("Micron Control Sys.")]</b> cut in above the cell slot."
+
+#define COMPANY_IZHEVSK "It has <b>[span_brown("Izhevsk Coalition")]</b> cut in above the magwell."
+#define COMPANY_TKACH "It has <b>[span_robot("Tkach Design Bureau")] stamped onto the reciever.</b>"
```

Comment:
```suggestion
#define COMPANY_TKACH "It has <b>[span_robot("Tkach Design Bureau")]</b> stamped onto the receiver."
```
Noticed this in the screenshot. Also according to autocorrect, receiver is spelled wrong everywhere in this file apparently? "I before E, **except after C**" or whatever

Reply:
What I'll do, since this can be applied to any atom, is just remove specific weapons parts mentions. A simple "It has x company name stamped on it" would do fine.